src/engines/kotorbase/module.cpp

```diff
@@ -179,6 +180,26 @@ Creature *Module::getPC() {
 	return _pc;
 }
 
+void Module::switchPC(unsigned int index) {
+	if (!_partyController.isAvailableCreature(index))
+		throw Common::Exception("%u is not an available npc id", index);
+
+	Creature *newPC = createCreature(_partyController.getAvailableNPCTemplate(index));
+	ObjectMan.registerObject(newPC);
+	_partyController.addPartyMember(index, newPC);
+	int oldLeaderID = _partyController.getPartyLeaderID();
+
+	Creature *oldPC = _partyController.getPartyLeader();
```

Comment:
Player character will not always be the party leader. Maybe we should modify `Module::_pc` instead?